sidecar/internal/service/reclaimspace.go

```
@@ -61,13 +68,34 @@ func (rs *ReclaimSpaceServer) ControllerReclaimSpace(
 	pvName := req.GetPvName()
 	klog.Info(pvName)
 
-	// TODO: get following params using kubeClient and pvName
+	pv, err := rs.kubeClient.CoreV1().PersistentVolumes().Get(ctx, pvName, metav1.GetOptions{})
+	if err != nil {
+		klog.Errorf("Failed to get pv: %v", err)
```

need to return an error, not only log it